config/url_components/minecraft.urls.json

```diff
@@ -0,0 +1,6 @@
+{
+  "baseTemplate": "https://www.minecraft.net/$lang/$article",
+  "components": [
+    { "$lang": "es-es", "$article": "community" }
+  ]
+}
```

Comment:
💅 nit: you should set up your vs code to enforce a new line at the end of files:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files